installation/kustomization.yaml

```diff
@@ -0,0 +1,57 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+- CONFIG_SYNC_MANIFEST
+# [WEBHOOK] - To install admission-webhook, uncomment the following line
+#- ADMISSION_WEBHOOK_MANIFEST
+
+patches:
+# [RESOURCES] - To adjust resource requests/limits, uncomment the following section and set the desired resources
+#- patch: |-
+#    - op: add
+#      path: "/spec/template/spec/containers/0/resources"
```

Comment:
The ordering of containers matters now. Do we have some test coverage to capture this to make sure we don't break users by accident?

Reply:
Yes, it's covered by the test case included in this commit.